content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -66,6 +66,124 @@ for the list of supported keys in `constraints` and `properties`.
     If you refresh the page several times, you should see different versions of reviews shown in the product page,
     presented in a round robin style (red stars, black stars, no stars)
 
+## RBAC policy permissive mode
+
+RBAC permissive mode is for testing a new RBAC policy works as expected before
+rolling out to production. This sections shows how RBAC permissive mode is used.
+
+### Permissive mode for global RBAC configuration
+
+Before you start, please make sure that you have finished [preparation task](#before-you-begin).
+
+1.  Before enabling RBAC in production, apply the global RBAC configuration in
+    permissive mode.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["default"]
+      enforcement_mode: PERMISSIVE
```

Comment:
enforcement_mode => mode?

Reply:
[RbacConfig](https://github.com/istio/api/blob/master/rbac/v1alpha1/rbac.proto#L257) already had a ```mode``` field, so have to put ```enforcement_mode``` here.

Reply:
Then let's use "enforcement_mode" in both per-policy and global RbacConfig? Consistent terminology is preferred.

Reply:
agree it's a bit confusing to better to align; I prefer to do it in follow cleanup PR, since rename will cause casanding change in different repo(api/istio/doc, etc)